main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/ONSdigital/dp-dimension-extractor/config"
+	"github.com/ONSdigital/dp-dimension-extractor/service"
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/s3"
+)
+
+func main() {
+	cfg := config.Get()
+
+	log.Namespace = "dp-dimension-extractor"
+
+	envMax, err := strconv.ParseInt(cfg.KafkaMaxBytes, 10, 32)
+	if err != nil {
+		log.ErrorC("encountered error parsing kafka max bytes", err, nil)
+	}
+
+	consumerGroup, err := kafka.NewConsumerGroup(cfg.Brokers, cfg.InputFileAvailableTopic, "input-file-available", cfg.InputFileAvailableOffset)
```

Comment:
Should the group name be configurable as well? Is it likely to change at a similar rate to topics - if the topic changes would the group name still make sense?